src/main/java/tbank/homework2/models/City.java

```diff
@@ -1,16 +1,15 @@
 package tbank.homework2.models;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.Setter;
-import lombok.ToString;
+import lombok.*;
 
 @Getter
 @Setter
 @ToString
+@EqualsAndHashCode
```

Comment:
Можно 
```suggestion
@Data
```
вместо @Getter
 @Setter
 @ToString
 @EqualsAndHashCode